chain/chain/src/test_utils/kv_runtime.rs

```diff
@@ -706,6 +707,13 @@ impl EpochManagerAdapter for MockEpochManager {
         Ok(chunk_producers[index].account_id().clone())
     }
 
+    fn get_validator_mandates_config(
```

Comment:
I see your point in the PR description, but now I wonder if AssignmentWeight should just be a Balance instead of a struct of number of full seats plus a partial balance. Then we wouldn't need to multiply it again to calculate anything. What do you think?

We can do that in a separate PR.

Reply:
Hmm, that makes sense... Why really keep all these seat numbers and partial seats etc.... We just need the overall balance

Reply:
Separate PR it is!